ckanext/canada/templates/internal/user/edit_user_form.html

```diff
@@ -33,9 +33,9 @@
                   attrs={'autocomplete': 'off', 'class': 'form-control'}
                   ) }}
 
-    {{ form.input('password1', type='password', label=_('Password'), id='field-password', value=data.password1, error=errors.password1, classes=['control-medium'], attrs={'autocomplete': 'off', 'class': 'form-control'} ) }}
+    {{ form.input('password1', type='password', label=_('New Password'), id='field-password', value=data.password1, error=errors.password1, classes=['control-medium'], attrs={'autocomplete': 'off', 'class': 'form-control'} ) }}
 
-    {{ form.input('password2', type='password', label=_('Confirm Password'), id='field-password-confirm', value=data.password2, error=errors.password2, classes=['control-medium'], attrs={'autocomplete': 'off', 'class': 'form-control'}) }}
+    {{ form.input('password2', type='password', label=_('Confirm New Password'), id='field-password-confirm', value=data.password2, error=errors.password2, classes=['control-medium'], attrs={'autocomplete': 'off', 'class': 'form-control'}) }}
```

Comment:
Are these aligning with upstream strings or were they requested by our side?

Reply:
@wardi neither, just semantic labels. Or do you think the legend of the field set is fine enough? Or did you want this suggested upstream?

Reply:
I ask because if it isn't upstream then we need to provide translations in our extension. Usually I leave the strings alone unless the business team asks for a change, IIRC they are in the process of reviewing the site now so it could be left to them.

Reply:
@wardi template changes have been reverted for the password labels.